src/shutdown/shutdown.c

```diff
@@ -379,6 +378,18 @@ int main(int argc, char *argv[]) {
                                           watchdog_device);
         }
 
+        watchdog_usec = getenv("WATCHDOG_USEC");
+        if (watchdog_usec) {
+                usec_t usec;
+
+                r = parse_time(watchdog_usec, &usec, 1);
```

Comment:
hmm, so this is actually not really a time expression, i.e. it's not supposed to understand "50min 20s", but should just be a plain usec in decimal string, i.e. use safe_atou64() to parse this, see how sd_watchdog_enabled() does this.

Reply:
OK